install-deps.sh

```diff
@@ -0,0 +1,6 @@
+wget http://www.cmake.org/files/v2.8/cmake-2.8.12.tar.gz
+tar xzf cmake-2.8.12.tar.gz
+cd cmake-2.8.12
+cmake .
+make
+sudo make install
```

Comment:
You probably don’t need to be root to install this package, just set its prefix to something like `~/.local`.

In Citra we download the binary instead, to make the build shorter, maybe you could take inspiration from this script: https://github.com/citra-emu/citra/blob/master/.travis-deps.sh